### ISSUE TITLE
Inconsistencies between filters available in tN vs tWL

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/tc-create-app#790]
Issue:
---
Remove ID and Occurrence and Reference fields. Note: filtering on chapter has been broken out into issue #800. Alphabetical sort broken out into issue #801

DoD:
---
ID, Occurrence, and Reference aren't shown in the filter dropdown.

more info:
----
v1.2.0-rc.1 build 26-371f3c3

In tN only the Chapter and SupportReference columns are available for filtering.
![image.png](https://images.zenhubusercontent.com/56e1a9316caf56c9d38d8df5/79530759-f457-4a60-af56-02ff9ed637b7)

In twL all columns are available for filtering. I think Chapter would make more sense than Reference (even though there is not a Chapter column). I don't think there is much value to filtering on Occurrence. Since ID is unique, the search function is probably more appropriate for it. In order for filtering on Original Word and TWLink to be useful, the lists should be sorted alphabetically.
![image.png](https://images.zenhubusercontent.com/56e1a9316caf56c9d38d8df5/c21775bf-c709-47cf-8047-c63adcdb5ae4)

**Comment by mandolyte** [comment]
@birchamp @cckozie this phrase: "and origWords and TWLink are sorted alphabetically".

What does it mean?

**Comment by mandolyte** [comment]
This means to sort the filter lists...
from: https://www.npmjs.com/package/mui-datatables#api
found:

Name | Type | Default | Description
-- | -- | -- | --
sortFilterList | boolean | true | Enable/disable alphanumeric sorting of filter lists.
-- | -- | -- | --


So the default is true...

**Comment by mandolyte** [comment]
Status 2021-04-09:
Per the docs, sorting is the default. Nonetheless, I tried to add this option explicitly to no effect.

I then went to a normal TN resource and notice it doesn't sort them either. So:

- if we sort, then it will not be like TN.
- the sorting option doesn't appear to work; even tho sorting is the default, and when explicitly added has no effect.
- if we sort, the reference column will not be in scripture order, since "1:10" will sort before "1:2".

I added the explicit sort option in datatable-translatable in `Datatable.js`:

```js
  const _options = {...options, sortFilterList: true};
  return (<MUIDataTable ref={dataTableElement} columns={columns} options={_options} data={data} />);
```
